test/01_TestOracle.test.js

```diff
@@ -18,19 +18,18 @@ const GasMeasuredOracleWrapper = artifacts.require('GasMeasuredOracleWrapper')
 require('chai').use(require('chai-as-promised')).should()
 
 contract('Oracle pricing', (accounts) => {
-  const testPriceWAD = '250000000000000000000'              // TestOracle just uses WAD (18-dec-place) pricing
+  const testPriceWAD = '250000000000000000000' // TestOracle just uses WAD (18-dec-place) pricing
 
-  const chainlinkPrice = '38598000000'                      // Chainlink aggregator (AggregatorV3Interface) stores 8 dec places
+  const chainlinkPrice = '38598000000' // Chainlink aggregator (AggregatorV3Interface) stores 8 dec places
   const chainlinkPriceWAD = new BN(chainlinkPrice + '0000000000') // We want 18 dec places, so add 10 0s
 
-  const compoundPrice = '414174999'                         // Compound view (UniswapAnchoredView) stores 6 dec places
+  const compoundPrice = '414174999' // Compound view (UniswapAnchoredView) stores 6 dec places
   const compoundPriceWAD = new BN(compoundPrice + '000000000000') // We want 18 dec places, so add 12 0s
 
-  const ethDecimals = new BN(18)                            // See OurUniswapV2TWAPOracle
+  const ethDecimals = new BN(18) // See OurUniswapV2TWAPOracle
```

Comment:
Personally I would probably leave the spacing choices on these comments (and others) as they are...  I find them easier to pick out visually.  Do you think this is bad?

Reply:
I don't think it's bad, but it's the linter that enforces it. I could see what is the rule that forbids that and disable it if you want